### ISSUE TITLE
enableMasteryProgression

### ISSUE THREAD

**Issue by NekoXIII** [thepeacockproject/Peacock#318]
set `enableMasteryProgression` to `false` by default, as the overwhelming majority of users want items unlocked without having to go into a vewy scawy ini file, or put in any effort for that matter

long overdue :+1:

**Comment by AnthonyFuller** [comment]
We've committed to it now.

**Comment by RDIL** [comment]
I'm with Neko on this one - either we need to change the default, OR we need to clearly state on the first run something like **MASTERY IS ENABLED BY DEFAULT - TO GET ALL ITEMS, SEE THIS DOC PAGE: link**

**Comment by grappigegovert** [comment]
I was thinking about having an in-game popup the first time you boot peacock where you can choose which you want, maybe?

**Comment by RDIL** [comment]
That sounds ie an excellent compromise

**Comment by AnthonyFuller** [comment]
People are just going to skip past it without reading it, you can't win. v7 will introduce in-game option editing, so it'll be friendlier for users to edit it.

**Comment by RDIL** [comment]
Sure people will skip past it but we can add like a confirmation

**Comment by AnthonyFuller** [comment]
I mean at this point just edit the privacy policy.

**Comment by RDIL** [comment]
I mean if we give an in game option that says

"How do you want to play the game?"
> Give me all items and 100% progression (Peacock classic behavior)
> Let me unlock items (Official servers behavior)

I think that's clear enough

**Comment by AnthonyFuller** [comment]
Sure, for v7.

**Comment by thecatontheceiling** [comment]
I would much prefer if the default **wasnt** everything being unlocked by default, most people use peacock for it's other features not because of I am too lazy to grind 

maybe ask the user a question in the Start Server.cmd window or something? or setup something in-game if that's possible

**Comment by riisikumi** [comment]
Based on my three years using Peacock and having at least some type of moderator/admin status in the server, everything being unlocked is exactly the reason why most people download and use Peacock

**Comment by thecatontheceiling** [comment]
augh

**Comment by AnthonyFuller** [comment]
Wait for v8.

**Comment by VolkerSchlegel** [comment]
I agree with @thecatontheceiling and @grappigegovert, some kind of prompt would be best, we can still make a default, and make a prompt that tells people about it, especially if in-game option editing becomes a thing.
